packages/teams-js/src/internal/telemetry.ts

```diff
@@ -1,5 +1,17 @@
 import { debug as registerLogger, Debugger } from 'debug';
 
+import { UUID } from './uuidObject';
+
+// Each teamsjs instance gets a unique identifier that will be prepended to every log statement
+export const teamsJsInstanceIdentifier = new UUID();
+
+// Every log statement will get prepended with the teamsJsInstanceIdentifier and a timestamp
+const origFormatArgs = registerLogger.formatArgs;
```

Comment:
```suggestion
const originalFormatArgs = registerLogger.formatArgs;
```
I wasn't sure if it was "origin" or "original" (though one could argue that the code makes it obvious). Suggestion for clarity